app/controllers/admin/legislation/draft_versions_controller.rb

```diff
@@ -1,6 +1,8 @@
 class Admin::Legislation::DraftVersionsController < Admin::Legislation::BaseController
-  load_and_authorize_resource :process, class: "Legislation::Process"
-  load_and_authorize_resource :draft_version, class: "Legislation::DraftVersion", through: :process
+  include Translatable
+
+  load_and_authorize_resource :draft_version, class: "Legislation::DraftVersion", through: :process, prepend: true
+  load_and_authorize_resource :process, class: "Legislation::Process", prepend: true
 
```

Comment:
Notice I've added `prepend: true`, so that `@draft_version` is loaded before the `Translatable.delete_translations` callback is called (which uses `@draft_version` through the `#resource` method.

@javierm What do you think of this? The order-dependence of callback is not ideal, but it avoids re-loading the resource in the resource method. I do think it makes sense have a the `load_resource` calls execute first as a general rule.

Reply:
Should be addressed in consul#2875.